packages/compose/README.md

```diff
@@ -196,11 +196,13 @@ render in components in `useCallback`.
 
 _Parameters_
 
--   _args_ `...any`: Arguments passed to Lodash's `debounce`.
+-   _fn_ `TFunc`: 
+-   _wait_ `[number]`: 
+-   _options_ `[import('lodash').DebounceSettings]`: 
```

Comment:
The description for the parameters is missing

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32015/commits/7e1ee137db3eabef0244f7a938566abbe16bf496